api/app/controllers/spree/api/base_controller.rb

```diff
@@ -101,8 +101,8 @@ def error_during_processing(exception)
 
         error_notifier.call(exception, self) if error_notifier
 
-        render :text => { :exception => exception.message }.to_json,
-          :status => 422 and return
+        render text: { exception: exception.message }.to_json,
+          status: 422 and return
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use `&&` instead of `and`.